File: app/src/main/java/com/example/narthana/genpass/WordContract.java
package com.example.narthana.genpass;

import android.provider.BaseColumns;

/**
 * Created by narthana on 23/10/16.
 */

final class WordContract
{
    // prevent accidental instantiation
    private WordContract() {}

    final static class WordEntry implements BaseColumns
    {
        static final String TABLE_NAME = "words";
        static final String COLUMN_WORD = "word";
        static final String COLUMN_LEN = "length";
    }
}

File: app/src/main/kotlin/com/example/android/genpass/data/WordContract.kt
package com.example.android.genpass.data

import android.provider.BaseColumns

/**
 * Created by narthana on 23/10/16.
 */

object WordContract {
    object WordEntry: BaseColumns {
        const val _ID = BaseColumns._ID
        const val TABLE_NAME = "words"
        const val COLUMN_WORD = "word"
        const val COLUMN_LEN = "length"
    }
}
